messaging/src/main/scala/uk/ac/wellcome/messaging/worker/monitoring/CloudwatchMonitoringClient.scala

```diff
@@ -0,0 +1,21 @@
+package uk.ac.wellcome.messaging.worker.monitoring
+
+import grizzled.slf4j.Logging
+import uk.ac.wellcome.monitoring.MetricsSender
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class CloudwatchMonitoringClient(metricsSender: MetricsSender)
```

Comment:
minor: what's the difference between putting an implicit ExecutionContext at method-level vs class-level? I tend to prefer class-level for cleaner code throughout.

Reply:
👍 Thanks